tests/conftest_trades_usdt.py

```diff
@@ -0,0 +1,305 @@
+        strategy='StrategyTestV2',
+        timeframe=5,
+    )
+    o = Order.parse_from_ccxt_object(mock_order_usdt_4(), 'ETC/USDT', 'buy')
+    trade.orders.append(o)
+    return trade
+
+
+def mock_order_usdt_5():
+    return {
+        'id': 'prod_buy_3455',
+        'symbol': 'XRP/USDT',
+        'status': 'closed',
+        'side': 'buy',
+        'type': 'limit',
+        'price': 2.0,
+        'amount': 10.0,
+        'filled': 10.0,
+        'remaining': 0.0,
+    }
+
+
+def mock_order_usdt_5_stoploss():
+    return {
+        'id': 'prod_stoploss_3455',
+        'symbol': 'XRP/USDT',
+        'status': 'open',
+        'side': 'sell',
+        'type': 'stop_loss_limit',
+        'price': 2.0,
+        'amount': 10.0,
+        'filled': 0.0,
+        'remaining': 30.0,
+    }
+
+
+def mock_trade_usdt_5(fee):
+    """
+    Simulate prod entry with stoploss
+    """
+    trade = Trade(
+        pair='XRP/USDT',
+        stake_amount=20.0,
+        amount=10.0,
+        amount_requested=10.01,
+        fee_open=fee.return_value,
+        fee_close=fee.return_value,
+        open_date=datetime.now(tz=timezone.utc) - timedelta(minutes=12),
+        is_open=True,
+        open_rate=2.0,
+        exchange='binance',
+        strategy='SampleStrategy',
+        stoploss_order_id='prod_stoploss_3455',
+        timeframe=5,
+    )
+    o = Order.parse_from_ccxt_object(mock_order_usdt_5(), 'XRP/USDT', 'buy')
+    trade.orders.append(o)
+    o = Order.parse_from_ccxt_object(mock_order_usdt_5_stoploss(), 'XRP/USDT', 'stoploss')
+    trade.orders.append(o)
+    return trade
+
+
+def mock_order_usdt_6():
+    return {
+        'id': 'prod_buy_6',
+        'symbol': 'LTC/USDT',
+        'status': 'closed',
+        'side': 'buy',
+        'type': 'limit',
+        'price': 10.0,
+        'amount': 2.0,
+        'filled': 2.0,
+        'remaining': 0.0,
+    }
+
+
+def mock_order_usdt_6_sell():
+    return {
+        'id': 'prod_sell_6',
+        'symbol': 'LTC/USDT',
+        'status': 'open',
+        'side': 'sell',
+        'type': 'limit',
+        'price': 12.0,
+        'amount': 2.0,
+        'filled': 0.0,
+        'remaining': 2.0,
+    }
+
+
+def mock_trade_usdt_6(fee):
+    """
+    Simulate prod entry with open sell order
+    """
+    trade = Trade(
+        pair='LTC/USDT',
+        stake_amount=20.0,
+        amount=2.0,
+        amount_requested=2.0,
+        open_date=datetime.now(tz=timezone.utc) - timedelta(minutes=5),
+        fee_open=fee.return_value,
+        fee_close=fee.return_value,
+        is_open=True,
+        open_rate=10.0,
+        exchange='binance',
+        strategy='SampleStrategy',
+        open_order_id="prod_sell_6",
+        timeframe=5,
+    )
+    o = Order.parse_from_ccxt_object(mock_order_usdt_6(), 'LTC/USDT', 'buy')
+    trade.orders.append(o)
+    o = Order.parse_from_ccxt_object(mock_order_usdt_6_sell(), 'LTC/USDT', 'sell')
+    trade.orders.append(o)
+    return trade
```

Comment:
I didn't actually end up using these, but I left them in because I thought maybe they could be helpful. I can remove them from the PR also